### ISSUE TITLE
Save State screenshot thumbnails do not work with 'Ozone' menu driver

### ISSUE THREAD

**Issue by prataczak** [libretro/RetroArch#10620]
## Description
For a long time now, if configured in settings/saving/save state thumbnails, when you would select a save state slot in the quick menu, you would get a thumbnail of a screenshot, which is very useful if you're trying to avoid saving over a particular state saved previously.

### Expected behavior
You select a state slot in the 'quick menu,' and a thumbnail appears, indicating which save state you are getting ready to load or save over top of.

### Actual behavior

No thumbnail appears.

### Version/Commit
RetroArch 1.86, current stable

### Environment information

- OS: Windows 10

Please do not suggest that I simply switch back to the 'XMB' menu driver, obviously I am aware that I can do that. Ozone is the new default, it is what is going to be used by everyone moving forward, and I am simply trying to get the same functionality I had before, on an otherwise very nice interface.

**Comment by askotx** [comment]
Same here, using Ozone this feature it's not working since a long time ago, version 1.8.1 ~ 1.8.5, config Saving->Save state thumbnails: ON. No display of screenshot after saving state, but screenshot it's created.

OS: Linux, Raspbian, arm RPi 3

**Comment by klepp0906** [comment]
was just coming to post this.  its one of the few things that has me torn about swapping to ozone.  no save state thumbnails kills me lol

**Comment by gmgman** [comment]
Are the devs even reading these posts at all? 30 days and no response.

**Comment by Rangnarok123** [comment]
I am in 1.8.8, and only try to use Ozone just now.
It's not working indeed.

**Comment by raaskmus** [comment]
In 1.8.9 and I guess it's something they can't decide how to best implement? It's the one feature I want for Ozone that worked so well in XMB! "Save state thumbnail" has to come back in some form, even if small and/or tucked away. Otherwise I grew to like Ozone but I really want thumbnails for my states for "ease of save state access" so I'm switching back just to have that feature again :0

**Comment by soredake** [comment]
Hope this will be fixed.

**Comment by soredake** [comment]
Any progress on this?

**Comment by Tropxe** [comment]
I'd like to add my comment here as well. It's a useful feature and it's still not working as far as I can tell.

**Comment by andiandi13** [comment]
I was about to open an issue for that, thanks for posting it.
I want that too, very useful when you have 9 save states.

It doesn't work in RGUI, GLUI and  Ozone

**Comment by cgarz** [comment]
Still an issue in 1.9.3. It's been more than a year now without so much as a peep :disappointed:.

It really sucks to have anyone who wants this awesome feature forced to use xmb and only xmb.
At the very least it would be nice to have an indication that a state slot actually has a save in it.
As it is now I'm using thumbnails not just because they are extremely useful, but to know if there is even something there in a slot in the first place.

**Comment by loucurapt** [comment]
Yes, at the very least RetroArch should highlight the states that are actually in use (bold or something), and the "load state" menu should only show those states.
Also would be nice if it showed the date/time of those save states.
And thumbnails in ozone.

**Comment by eddlang** [comment]
I'm relatively new to RA and at first thought save state thumbnails were completely broken. Now I know it's an ozone issue.

Not having thumbnails detracts from this, otherwise very well put, UI.

I'm also in full agreement with @loucurapt. Trying to find a save state sometimes becomes where's waldo.

**Comment by askotx** [comment]
@jdgleaver could you lend a hand?

**Comment by jdgleaver** [comment]
This is not an easy thing to add... I can think about it, but no promises.

(I've also been meaning to overhaul save states completely for quite a while now, but everything else keeps getting in the way...)

**Comment by askotx** [comment]
Fair enough.

**Comment by auwsmit** [comment]
I think until savestate thumbnails are implemented in other menus, the option should be disabled and/or have an indication that it only works in the XMB menu for now... assuming that's not more trouble than it's worth.

Although it is hidden behind "advanced settings", I was still confused about why it didn't work and spent a bit trying to troubleshoot it before I found this issue, so if the option info had informed me that it only works in XMB it would've saved me some time and minor headache.

**Comment by andiandi13** [comment]
@auwsmit True.
But in another hand I'm surprised that it hasn't been implemented in other menus yet, that's seems so easy
